helm/reana/templates/reana-config.yaml

```diff
@@ -34,3 +34,10 @@ data:
     {{- end }}
     quota_enabled: {{ .Values.quota.enabled }}
     file_preview_size_limit: {{ int .Values.components.reana_ui.file_preview_size_limit }}
+    launcher_repositories:
```

Comment:
Let's call this `launcher_examples` to be consistent with the rest of the config